newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js

```diff
@@ -134,22 +141,102 @@ export default class LayerRenderer {
     );
   };
 
-  getInstanceWidth = (instance: gdInitialInstance) => {
+  getUnrotatedInstanceWidth = (instance: gdInitialInstance) => {
     if (instance.hasCustomSize()) return instance.getCustomWidth();
 
     return this.renderedInstances[instance.ptr]
       ? this.renderedInstances[instance.ptr].getDefaultWidth()
       : 0;
   };
 
-  getInstanceHeight = (instance: gdInitialInstance) => {
+  getUnrotatedInstanceHeight = (instance: gdInitialInstance) => {
     if (instance.hasCustomSize()) return instance.getCustomHeight();
 
     return this.renderedInstances[instance.ptr]
       ? this.renderedInstances[instance.ptr].getDefaultHeight()
       : 0;
   };
 
+  getInstanceOBB(instance: gdInitialInstance, bounds: Rectangle): Rectangle {
```

Comment:
Is this really an "oriented bounding box" that we're returning here? I think this should rather be called "getUnrotatedInstanceAABB" (i.e: the axis aligned bounding box of the unrotated instance) (but not getInstanceUnrotatedAABB, because by definition an AABB cannot be "rotated").